Source/SpaceParkAPI/SWAPI/Models/Person.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace SpaceParkAPI.SWAPI.Models
+{
+    public class Person
```

Comment:
```suggestion
    public record Person
```
A suggestion here is to use the keyword record introduced in C# 9.0
https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/records
This keyword is prefered when making model-type classes - value equality is autohandled for instance. This applies to all model classes in the project. :)

Reply:
Tack för förslaget!